README.md

```diff
@@ -292,7 +294,8 @@ Table of Contents
    * http://cloudinary.com - Image upload, powerful manipulations, storage, and delivery for sites and apps, with libraries for Ruby, Python, Java, PHP, Objective-C and more. Perpetual free tier includes 7500 images/month, 2gb storage, 5gb bandwidth.
    * https://plot.ly - graph and share your data. Free tier includes unlimited public files and 10 private files.
    * https://transloadit.com - Handles file uploads & encoding of video, audio, images, documents. Free for open source & other do-gooders. Commercial applications get the first GB free for testdriving.
-   * https://cartodb.com - create maps and geospatial APIs from your data and public data
+   * https://cartodb.com - Create maps and geospatial APIs from your data and public data
+   * https://zapier.com - Connect web apps to move data between them. Free 5 zap, each 15 min, 100 tasks/month.
```

Comment:
? https://zapier.com/status/
hundreds of API for integration, e.g. my soft save on box, mega, mail to xyz...